src/assets/translations/messages-en.json

```diff
@@ -431,7 +431,7 @@
       "success": {
         "encryptingWallet": "Encrypting your wallet... if your browser asks to store data in persistent storage, please click 'Allow'.",
         "header": "Wallet Connected",
-        "success": "Your wallet has been connected",
+        "success": "You can now close this window or it will auto-close shortly",
```

Comment:
you're definitely not up to date with develop

Reply:
how is this even possible, I was the one that introduced that change in the translations even 🤯

Reply:
wait a minute, something is wrong here

Reply:
fixed now lol